### ISSUE TITLE
Extremely large jump in cases in France from 2020-04-11 to 2020-04-12

### ISSUE THREAD

**Issue by jeanpaulrsoucy** [CSSEGISandData/COVID-19#2360]
Hi,

I was looking at the confirmed case time series for France, and there is a massive jump in cases between 2020-04-11 and 2020-04-12 (Easter Sunday), with cases going from 93790 to 120633. This is an increase of almost 27,000, compared to a few thousand during the preceding and following days. I thought it might be due to a holiday-related reporting delay, but other sources do not seem to have feature - they do not have the same spike.

**Comment by mackuba** [comment]
Yeah, there's been like 30 threads about it since then, run the search - they said they'd fix it, and then they didn't…

**Comment by traut21** [comment]
See https://github.com/CSSEGISandData/COVID-19/issues/2270

JHU does not show any professional reaction
